Makefile

```diff
@@ -92,7 +92,14 @@ generate-modular-docs: generate-docs ## Generate modular command-line reference
 	SRC_DIR=$$(pwd)/docs/commands DEST_DIR=$$(pwd)/dist go run ./cmd/modular-docs
 .PHONY: generate-modular-docs
 
+lint-lang:
+	git clone git@github.com:redhat-developer/app-services-go-linter
```

Comment:
`go install github.com/redhat-developer/app-services-go-linter`

Reply:
I could not make it work:

go install github.com/redhat-developer/app-services-go-linter
no required module provides package github.com/redhat-developer/app-services-go-linter; to add it:
        go get github.com/redhat-developer/app-services-go-linter

Reply:
It needs to be to the path to the main package:

```shell
go install github.com/redhat-developer/app-services-go-linter/cmd/app-services-go-linter@latest
```

Reply:
Nice. Fixing it